base_time_window/models/time_weekday.py

```diff
@@ -60,3 +60,20 @@ def unlink(self):
         result = super().unlink()
         self._get_id_by_name.clear_cache(self)
         return result
+
+    def _get_next_weekday_date(self, date_from=False, include_date_from=True):
+        """Returns the next Date matching weekday
+
+        :param date_from: Date object from which we start searching for next weekday
+        :param include_date_from: Allows to return date from if it's same weekday
+        :return Date object matching the weekday
+        """
+        self.ensure_one()
+        if not date_from:
+            date_from = fields.Date.today()
+        next_date = date_from
+        if next_date.weekday() == int(self.name) and not include_date_from:
```

Comment:
Can't we not just do
```suggestion
        if not include_date_from:
```

Reply:
Must be fixed in fd25b083bbeb86b47a4e771ca3652e4c4d441276 , IDK why but it seems I forgot to push it.